core/src/main/java/com/ritense/valtimo/autoconfigure/ValtimoAutoConfiguration.java

```diff
@@ -34,16 +34,11 @@
 import com.ritense.valtimo.repository.CamundaReportingRepository;
 import com.ritense.valtimo.repository.CamundaSearchProcessInstanceRepository;
 import com.ritense.valtimo.repository.CamundaTaskRepository;
+import com.ritense.valtimo.repository.UserSettingsRepository;
 import com.ritense.valtimo.security.permission.Permission;
 import com.ritense.valtimo.security.permission.TaskAccessPermission;
 import com.ritense.valtimo.security.permission.ValtimoPermissionEvaluator;
-import com.ritense.valtimo.service.AuthorizedUsersService;
-import com.ritense.valtimo.service.BpmnModelService;
-import com.ritense.valtimo.service.CamundaProcessService;
-import com.ritense.valtimo.service.CamundaTaskService;
-import com.ritense.valtimo.service.ContextService;
-import com.ritense.valtimo.service.ProcessPropertyService;
-import com.ritense.valtimo.service.ProcessShortTimerService;
+import com.ritense.valtimo.service.*;
```

Comment:
Don't use wildcard imports in production code. Change your settings in your IDE please.